### PR TITLE
multicast: fix compile failed on windows

### PR DESCRIPTION
**Pull request by jobo-zt**

### CODE REVIEW

File: modules/multicast/multicast.c [hunk 1]
```diff
@@ -6,7 +6,9 @@
  * Copyright (C) 2021 Commend.com - c.huber@commend.com
  */
 
+#ifdef HAVE_GETIFADDRS
 #include <net/if.h>
+#endif
```
Comment (cspiel1) [inline modules/multicast/multicast.c:11]: @cHuberCoffee  Not sure. Can't we simply drop this?

Can't find this include in baresip

Comment (maximilianfridrich) [reply modules/multicast/multicast.c:11]: It is needed for IPv6 multicast.

**Comment by maximilianfridrich** [comment]
Thank you for the fix! Could you please extend the `#ifdef` block to also include the `#undef`s below?

File: modules/multicast/multicast.c [hunk 2]
```diff
@@ -6,10 +6,12 @@
  * Copyright (C) 2021 Commend.com - c.huber@commend.com
  */
 
-#include <net/if.h>
 #undef LIST_INIT
 #undef LIST_FOREACH
```
Comment (maximilianfridrich) [inline modules/multicast/multicast.c:10]: I meant more like this:

```c
#ifdef HAVE_GETIFADDRS
#include <net/if.h>
#undef LIST_INIT
#undef LIST_FOREACH
#endif
```

**Comment by maximilianfridrich** [comment]
LGTM

**Pull request merged**